css.md

```diff
@@ -205,3 +205,70 @@ Variables are a great way to make your css more manageable, so you're not editin
       background-color: var(--primary-color);
     }
 ```
+
+### 9. Animations
```

Comment:
Can you make this as a table, that way will be concise

Reply:
sure